packages/create-sitecore-jss/src/init-runner.ts

```diff
@@ -25,9 +26,8 @@ export const initRunner = async (initializers: string[], args: BaseArgs) => {
       args.silent || console.log(chalk.cyan(`Initializing '${init}'...`));
       const response = await initializer.init(args);
 
-      appName = response.appName;
+      appNames.add(response.appName);
```

Comment:
Need to add unit tests to cover the scenario when two initializers produce two standalone apps, so there will be two unique app names